InjectionIII/AppDelegate.mm

```diff
@@ -55,7 +55,7 @@ - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
         : NSControlStateValueOff;
 
     [self setMenuIcon:@"InjectionIdle"];
-#if 0
+#if 01
```

Comment:
Do we need this statement now?

Reply:
Not really. In fact the code is getting pretty untidy. I’ll take it out.